lib/utility/translateWhereToQuery.js

```diff
@@ -9,7 +9,7 @@ exports.translateWhereToQuery = function (where, values) {
   for (var key in where) {
     if (where.hasOwnProperty(key) && where[key] !== undefined) {
       whereQuery += whereQuery === '' ? ' WHERE' : ' AND'
-      whereQuery += ' ' + key
+      whereQuery += ' ' + this.escapeKeys([key])
```

Comment:
also, I'm not sure, but here I think it should be `this.escapeKeys([key])[0]`, because the function `escapeKeys` returns array, so it's gonna be more explanatory if it's `this.escapeKeys([key])[0]`, instead of `this.escapeKeys([key])`.